src/strpkg/collect.nim

```diff
@@ -86,8 +86,10 @@ proc count(A: Record, bounds:Bounds): int =
     return 0
 
   if read_left < 0: read_left = 0
-
-  return dna[read_left..<read_right].count(bounds.repeat)
+  let S = dna[read_left..<read_right]
```

Comment:
@hdashnow you can revert this section to test effect

Reply:
i already forgot why we did this section. should add a comment.

Reply:
I don't remember either...

Reply:
So I think this boils down to:
```if repeats_in_reads < int(read_length * 0.7 / repeat_length):
    repeats_in_reads```